docs/sources/datasources/tempo/configure-tempo-data-source.md

```diff
@@ -141,19 +142,33 @@ There are two ways to configure the trace to metrics feature:
 - Use a basic configuration with a default query, or
 - Configure one or more custom queries where you can use a [template language](ref:variable-syntax) to interpolate variables from the trace or span.
 
-### Simple config
+Refer to the Trace to metrics configuration options section to learn about the available options.
+
+### Set up a simple configuration
 
 To use a simple configuration, follow these steps:
 
 1. Select a metrics data source from the **Data source** drop-down.
+1. Optional: Change the **Span start time shift** and/or **Span end time shift**. The default start time shift is -2 minutes and 2 minutes for end time shift.
```

Comment:
https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanaandor

```suggestion
1. Optional: Change the **Span start time shift** and **Span end time shift**. The default start time shift is -2 minutes and 2 minutes for end time shift.
```

Reply:
Same as before. In this case, it's an option where someone could change one or both. I'm not sure if someone would want to have them as different numbers, but technically they could. I'll see if there is a clearer way to message this and will check with a dev to see if both of these have to use the same value (i.e., -3/+3) or if different values (i.e., -3/+5) are valid.

Reply:
Perhaps it's most clear to have them as two optional steps where each is independently described?

Reply:
I believe the intention of the style rule is that being explicit is preferable to being succinct.

Reply:
I can do that. Thank you for the suggestion.